hw/ip/aes/data/aes.hjson

```diff
@@ -266,7 +266,7 @@
       ]
       tags: [// Updated by the HW.
              // Updates based on writes to other regs.
-             // No reset but sync clear with random data.
+             // Upon reset internal operations are triggered that clear these registers with random data.
```

Comment:
Perhaps I am missing the context here for the hjson file but seems like "with random data" is not correct here, as the clearing of data_out is with zero according to the RTL below.

Reply:
I see, this is probably not fully clear. It's correct that the register is reset to zero. But at the same time, the trigger register is reset to a value that causes the AES unit to first reseed the PRNG and then clear all registers with random data. For this reason, automated CSR checks that try to check the reset value (defined as value after reset and before software has written these regs) will fail. The register needs to be excluded from such tests.

I will clarify the comment.